osu.Game.Rulesets.Mania/Difficulty/Skills/Strain.cs

```diff
@@ -4,98 +4,281 @@
 #nullable disable
 
 using System;
-using osu.Framework.Utils;
 using osu.Game.Rulesets.Difficulty.Preprocessing;
 using osu.Game.Rulesets.Difficulty.Skills;
 using osu.Game.Rulesets.Mania.Difficulty.Preprocessing;
 using osu.Game.Rulesets.Mods;
 
+/* Rules of Strain Calculation
+ * 1) The current strain value can only depend on past notes
```

Comment:
"can only depend on past notes" - as opposed to what? As in, what does it _not_ depend on that people might think it does?

Reply:
As opposed to being an independent measure of difficulty.

Take for example
Notes: A B C had strains 1 2 3
If they are independent, then the evaluated strain should be 1 3 6
else, the evaluated strain should be 1 2 3 (as they consider dependencies already)

I do agree that the wording is vague, I'll amend this section